Makefile.am

```diff
@@ -159,14 +159,14 @@ src/Symbols.static: $(SYMBOL_SCRIPT)
 AUTOHEADER=echo
 
 ########
-# doc/ #
+# docs/ #
 ########
 
 if FULL_SUITE
-dist_html_DATA = doc/index.html doc/libsndfile.jpg doc/libsndfile.css doc/print.css doc/api.html \
-	doc/command.html doc/bugs.html doc/formats.html doc/sndfile_info.html doc/new_file_type.HOWTO \
-	doc/win32.html doc/FAQ.html doc/lists.html doc/embedded_files.html doc/octave.html \
-	doc/tutorial.html
+dist_doc_DATA = docs/index.md docs/libsndfile.jpg docs/libsndfile.css docs/print.css docs/api.md \
```

Comment:
what's the reason for changing these?

Reply:
`doc/` directory to `docs/`? or HTML source to Markdown?

Reply:
oh, I just realised that you replaced them with Markdown.